src/functionalTest/resources/request/casedata/ccd-callback-caseworker-issue-sole-citizen-application-about-to-submit.json

```diff
@@ -13,10 +13,11 @@
     "Country": "United Kingdom"
   },
   "applicant2Address": {
-    "AddressLine1": "Parc de la Ciutadella",
```

Comment:
is all this data needed for this specific callback? might be easier to manage if you just restrict it to the data used in the callback

Reply:
im not sure as I just updated the address to use the UK one and created a similar test with overseas address.